theSrc/internal_www/content/examples/bubbleplot.html

```diff
@@ -1,79 +1,39 @@
 <!doctype html>
 <html>
 <head>
-    <link href='https://fonts.googleapis.com/css?family=Arvo:400,700italic,700,400italic' rel='stylesheet' type='text/css'>
-    <link href='/styles/internal_www.css' rel='stylesheet' type='text/css'>
-    <!--Scripts below used by rhtmlLabeledScatter-->
-    <script src="../../internal_www/external/lodash.min.js"></script>
-    <script src="../../internal_www/external/jquery.min.js"></script>
-    <script src="../../internal_www/external/d3.min.js"></script>
-    <script src="../../internal_www/external/random.min.js"></script>
-    <script src="../../internal_www/external/bluebird.min.js"></script>
-    <script src="../../internal_www/external/bignumber.min.js"></script>
+  <link href='https://fonts.googleapis.com/css?family=Arvo:400,700italic,700,400italic' rel='stylesheet' type='text/css'>
+  <link href='/styles/internal_www.css' rel='stylesheet' type='text/css'>
+  <!--Scripts below used by rhtmlLabeledScatter-->
+  <script src="../../internal_www/external/lodash.min.js"></script>
+  <script src="../../internal_www/external/jquery.min.js"></script>
+  <script src="../../internal_www/external/d3.min.js"></script>
+  <script src="../../internal_www/external/random.min.js"></script>
+  <script src="../../internal_www/external/bluebird.min.js"></script>
+  <script src="../../internal_www/external/bignumber.min.js"></script>
 </head>
 <body>
-
-
-<section>
+  <section>
     <h2>Bubble plot</h2>
```

Comment:
this is an example of how to use `renderContentPage.js` to make the content html files cleaner and more consistent